dist/sordid-dragon.jquery.js

```diff
@@ -2,12 +2,12 @@
 // Copyright Poll Everywhere
 // Paul Cortens & Mike Foley
 // https://github.com/polleverywhere/sordid-dragon
-// Version 1.3.0
+// Version 1.3.1
```

Comment:
It looks like the generated version is out of date with the CoffeeScript file: https://github.com/polleverywhere/sordid-dragon/pull/2/files#diff-29432394491edfdbf090198d60791285R5